### ISSUE TITLE
[Bug] "Failed to resolve dependencies" when updating

### ISSUE THREAD

**Issue by TheFrenchGhosty** [iv-org/invidious#1863]
**Describe the bug**
"Failed to resolve dependencies" is displayed when updating, even though Crystal 0.36 is indeed installed.

**Steps to Reproduce**

Update to latest master

**Logs**

```
Unable to satisfy the following requirements:

- `crystal (>= 0.35.0, < 2.0.0)` required by `pg 0.23.1`
- `crystal (>= 0.35.0, < 2.0.0)` required by `sqlite3 0.18.0`
- `crystal (~> 0.35, >= 0.35.0)` required by `kemal 0.27.0`
- `crystal (< 1.0.0)` required by `pool 0.2.3`
- `crystal (~> 0.34, >= 0.34.0)` required by `protodec 0.1.3`
- `crystal (~> 0.36, >= 0.36.1)` required by `lsquic 2.23.1`
- `crystal (~> 0.35, >= 0.35.0)` required by `db 0.10.0`
- `crystal (< 1.0.0)` required by `radix 0.3.9`
Failed to resolve dependencies, try updating incompatible shards or use --ignore-crystal-version as a workaround if no update is available.
```

**Screenshots**

N/A

**Additional context**

Crystal 0.36 is installed:

```
# crystal --version
Crystal 0.36.0 (2021-01-26)
```

Any idea @saltycrys ?

**Comment by thatHguy** [comment]
Can confirm this bug on my Debian install.

```
$ shards update && shards install
Resolving dependencies
Fetching https://github.com/will/crystal-pg.git
Fetching https://github.com/crystal-lang/crystal-sqlite3.git
Fetching https://github.com/kemalcr/kemal.git
Fetching https://github.com/ysbaddaden/pool.git
Fetching https://github.com/omarroth/protodec.git
Fetching https://github.com/iv-org/lsquic.cr.git
Fetching https://github.com/crystal-lang/crystal-db.git
Fetching https://github.com/luislavena/radix.git
Unable to satisfy the following requirements:

- crystal (>= 0.35.0, < 2.0.0) required by pg 0.23.1
- crystal (>= 0.35.0, < 2.0.0) required by sqlite3 0.18.0
- crystal (~> 0.35, >= 0.35.0) required by kemal 0.27.0
- crystal (< 1.0.0)` required by pool 0.2.3
- crystal (~> 0.34, >= 0.34.0) required by protodec 0.1.3
- crystal (~> 0.36, >= 0.36.1) required by lsquic 2.23.1
- crystal (~> 0.35, >= 0.35.0) required by db 0.10.0
- crystal (< 1.0.0)` required by radix 0.3.9
Failed to resolve dependencies
```


```
$ crystal -v 
Crystal 0.35.1 [5999ae29b] (2020-06-19)

LLVM: 8.0.0
Default target: x86_64-unknown-linux-gnu
```

**Comment by markozajc** [comment]
Updating works for me, but I did have to change the Crystal repository location on Debian, as they have moved it:
```$ crystal -v 
Crystal 0.36.1 [c3a3c1823] (2021-02-02)

LLVM: 10.0.0
Default target: x86_64-unknown-linux-gnu
```
Relevant: https://crystal-lang.org/install/on_debian/
I'm also using the [Invidious-Updater](https://github.com/tmiland/Invidious-Updater) script, and am on Debian Testing, if that makes any difference.

**Comment by TheFrenchGhosty** [comment]
Solved by updating to Crystal 0.36.1.
